docs/blueprints.rst

```diff
@@ -20,9 +20,10 @@ Why Blueprints?
 
 Blueprints in Flask are intended for these cases:
 
-* Factor an application into a set of blueprints.  This is ideal for
-  larger applications; a project could instantiate an application object,
-  initialize several extensions, and register a collection of blueprints.
+* Factor an application into a modular set of related logical components,
+  ideal for larger applications. A project could instantiate an application
+  object , register a collection of blueprints on the app. This makes
```

Comment:
`object ,` has a trailing space

Reply:
How about this:
`Architect a Flask app into a modular set of logically related components (ideal for larger apps). A project could instantiate a Flask app object, and then register a collection of blueprints on this instance. The blueprints can be reused, meaning your code can be made easier to maintain.`